src/controllers/controller.line.js

```diff
@@ -47,9 +47,23 @@ module.exports = DatasetController.extend({
 		var options = me.chart.options;
 		var lineElementOptions = options.elements.line;
 		var scale = me.getScaleForId(meta.yAxisID);
-		var i, ilen, custom;
 		var dataset = me.getDataset();
 		var showLine = lineEnabled(dataset, options);
+		var index = me.index;
+		var model, i, ilen, custom;
+
+		var LINE_ELEMENT_OPTIONS = {
+			tension: 'lineTension',
+			backgroundColor: 'backgroundColor',
+			borderColor: 'borderColor',
+			borderWidth: 'borderWidth',
+			borderCapStyle: 'borderCapStyle',
+			borderDash: 'borderDash',
+			borderDashOffset: 'borderDashOffset',
+			borderJoinStyle: 'borderJoinStyle',
+			fill: 'fill',
+			cubicInterpolationMode: 'cubicInterpolationMode'
+		};
```

Comment:
We should not use a map for the line options but instead, `tension` should be a special case (independently resolved [as here](/chartjs/Chart.js/blob/master/src/controllers/controller.bubble.js#L167)), else we will evaluate the same value twice for all properties.

IMO, the point prefixed options logic is a special case which shouldn't be use as common implementation when resolving element options.